src/planet-a/PlanetGeometryA.cpp

```diff
@@ -268,80 +205,161 @@ struct VertexToSubdiv
     unsigned m_vrtxIndex;
 };
 
-void PlanetGeometryA::chunk_add(trindex_t t)
+// temporary
+float debug_stupid_heightmap(Vector3 pos)
 {
-    SubTriangle& tri = m_icoTree->get_triangle(t);
-
-    trindex_t min = std::max<trindex_t>(m_triangleChunks.size(),
-                                        m_icoTree->triangle_count());
-    m_triangleChunks.resize(min, {gc_invalidChunk, 0, gc_invalidTri, 0, 0});
+    float raise = 0; // add together a bunch of sin waves for terrain
+    raise += (std::sin(pos.x() / 128.0f) + std::sin(pos.y() / 128.0f) + std::sin(pos.z() / 128.0f)) * 64.0f;
+    raise += (std::sin(pos.x() / 500.0f) + std::sin(pos.y() / 500.0f) + std::sin(pos.z() / 500.0f)) * 128.0f;
+    raise += (std::sin(pos.x() / 720.0f) + std::sin(pos.y() / 720.0f) + std::sin(pos.z() / 720.0f)) * 512.0f;
+    return raise * 0.0f; // remove 0.0f for fun on the moon
```

Comment:
So always return 0?

Reply:
This code was written with a moon-sized planet in mind. The small planet in the 'simple' test universe gets messed up with this function. It displaces the terrain too much, making the planet into a mess, so it was disabled by adding the `* 0.0f`.